docs/changelog/1700.feature.rst

```diff
@@ -0,0 +1 @@
+Allow {/} to refer to os.sep. - by :user:`jayvdb`
```

Comment:
This is https://github.com/tox-dev/tox/pull/1701

Without this, the tests need to do hacks to convert `/` to `\` on Windows.  It gets messy because the bugs related to this are about real native paths, including constructed path based on preset paths.